docs/reference/glossary.asciidoc

```diff
@@ -4,97 +4,121 @@
 
 [glossary]
 [[glossary-analysis]] analysis ::
-
-Analysis is the process of converting <<glossary-text,full text>> to
-<<glossary-term,terms>>. Depending on which analyzer is used, these phrases:
++
+--
+// tag::analysis-def[]
+Analysis is the process of converting link:#glossary-text[full text] to
+link:#glossary-term[terms]. Depending on which analyzer is used, these phrases:
 `FOO BAR`, `Foo-Bar`, `foo,bar` will probably all result in the
 terms `foo` and `bar`. These terms are what is actually stored in
 the index.
-+
-A full text query (not a <<glossary-term,term>> query) for `FoO:bAR` will
+
+A full text query (not a link:#glossary-term[term] query) for `FoO:bAR` will
 also be analyzed to the terms `foo`,`bar` and will thus match the
 terms stored in the index.
-+
+
 It is this process of analysis (both at index time and at search time)
 that allows Elasticsearch to perform full text queries.
-+
-Also see <<glossary-text,text>> and <<glossary-term,term>>.
 
-[[glossary-cluster]] cluster ::
+Also see link:#glossary-text[text] and link:#glossary-term[term].
```

Comment:
For interest's sake is there a reason for using the link macro here instead of the xref macro or double angled brackets (per https://asciidoctor.org/docs/user-manual/#internal-cross-references)?

Reply:
Thanks for asking!

I switched back to the double angled bracket with 6239749.

I think I got myself confused. I was thinking that any internal xrefs would need to be updated to links... but that wouldn't apply to glossary anchors.